js/views/settingsVw.js

```diff
@@ -236,6 +239,22 @@ module.exports = pageVw.extend({
         placeholder_text_multiple: window.polyglot.t('chosenJS.placeHolderTextMultiple')
       });
 
+      var connectedPeers = "";
+      $.ajax({
+        url: self.serverUrl+ "routing_table",
+        success: function(data){
+          self.$el.find('span.js-numConnectedPeers').text(data.length);
+          data.forEach(function (peer) {
+            connectedPeers += (peer['ip'] + ":" + peer['port'] + "\n");
+          })
+          self.$el.find('span.js-connectedPeers').text(connectedPeers);
+          self.$el.find('.js-connectedPeers').hide();
```

Comment:
Could we please chain these two lines, otherwise you're making an unnecessary second query of the DOM?

```
self.$el.find('span.js-connectedPeers')
  .text(connectedPeers)
  .hide();
```